packages/editor/src/server/index.tsx

```diff
@@ -27,7 +27,8 @@ async function asyncMain() {
 
   const clientSettings = {
     apiURL: process.env.API_URL,
-    peerByDefault: process.env.PEER_BY_DEFAULT?.toLowerCase() === 'true'
+    peerByDefault: process.env.PEER_BY_DEFAULT?.toLowerCase() === 'true',
+    imgMinSizeToCompress: process.env.IMG_MIN_SIZE_TO_COMPRESS
```

Comment:
To avoid `imgMinSizeToCompress` to be `string | undefined` i would add a default value here if the environment variable is undefined. Also i suggest to use `number` instead of `string`. Since we're passing those settings via JSON to the client we can already use the proper type.

So this line could look like:
`imgMinSizeToCompress: parseInt(process.env.IMG_MIN_SIZE_TO_COMPRESS) || 10`

Reply:
yeah. I improved it with an ternary operator since parseInt() doesn't like undefined that much ;)